src/util/graphic.js

```diff
@@ -287,6 +287,12 @@ function singleEnterEmphasis(el) {
 
     if (useHoverLayer) {
```

Comment:
Why not in SVG, useHoverLayer is always `false`.
It's a little confused that the behavior of calling addHover and getting a null